### DISCUSSION TITLE
Fix pervasive mismatch between TOSCA types and YAML types in spec

### DISCUSSION THREAD

**Discussion by tliron** [oasis-tcs/tosca-specs#232] [Ideas]
Throughout the spec we detail the keyname tables in Markdown similar to this:

```
|Keyname|Mandatory|Type|Description|
| :---- | :------ | :---- | :------ |
|`tosca_definitions_version`|yes|[string](#string)|Defines the [version of the TOSCA specification](#tosca-definitions-version) used in this TOSCA file.|
|`description`|no|[string](#string)|Declares a [description](#description) for this TOSCA file and its contents.|
|`metadata`|no|[map](#map) of YAML values|Defines a section used to declare additional [metadata](#metadata) information. Domain-specific TOSCA profile specifications may define keynames that are mandatory for their implementations.|
```

Note how the "Type" column features links to sections in TOSCA data types section.

Unfortunately, this is wrong and misleading. The keynames do NOT accept TOSCA data types, but rather YAML values. For example, our TOSCA "map" has key_schemas and entry_schemas that have nothing to do with YAML. A user clicking on such a link to learn more will be led down the wrong path.

If folk agree with me that this is misleading, let's turn this discussion into an issue and I will make the fixes.

Also note that lists in YAML are called "seqs" and should be fixed accordingly.

**Comment by pmjordan** [comment]
Thanks for pointing out that error
